src/frontend/lib/query-persister.ts

```
@@ -1,12 +1,10 @@
-// src/lib/query-persister.ts
-import { get, set, del } from 'idb-keyval'
-
-import { IDB_KEYS } from './storage-keys'
-
 import type {
   PersistedClient,
   Persister
 } from '@tanstack/query-persist-client-core'
+import { get, set, del } from 'idb-keyval'
```

ESLint

[error] Unable to resolve path to module 'idb-keyval'.

(import-x/no-unresolved)